docs/architecture/overview.md

```diff
@@ -0,0 +1,163 @@
+# Overview
+
+The components that make up a running test are simple from the outside.
+
+The idea behind system tests is that we can share the tests for a given feature across implementations.
+
+Enabling a feature within system tests might go like this:
+ - Add the enabling environment variable to the shared docker-compose
+ - Add a test asserting the environment variable affected traces as expected
+ - Build and run for each language to verify the tests run and behave as expected
+ - Create the pull request in system-tests
+
+However, there are many scenarios where a test may not be so simple to implement.
+
+This document aims to give a working understanding of the parts of system-tests, and how to troubleshoot them.
+
+## What are the components of a running test?
```

Comment:
not sure if that's the right section, but it's probably also worth mentioning how tests can be integrated to the CI/CD process and where to find the results (link to the dashboard)

Reply:
I will do so.

Reply:
I added a section about how to test specific tracer instances but would like to do a follow up if that's okay.